src/main/java/org/folio/rs/domain/entity/AccessionQueueRecord.java

```diff
@@ -34,4 +34,34 @@
   private String instanceTitle;
 
   private String instanceAuthor;
+
+  private String instanceCreator;
```

Comment:
I think creator/author/contributor is the same persons. Can we create list of contributors like https://github.com/folio-org/mod-inventory/blob/f70c37d435594e9a6d108e66b879448f466108b2/ramls/item.json#L35?

Reply:
The instance creator field was removed.